docs/user-guide.md

```diff
@@ -256,7 +256,7 @@ file "redis-rc.yaml" created
 file "web-rc.yaml" created
 ```
 
-The `*-rc.yaml` files contain the Replication Controller objects. If you want to specify replicas (default is 1), use `--replicas` flag: `$ kompose convert --rc --replicas 3`
```

Comment:
You need to change `--dc` to `--deployment-config` and `--rc` to `--replication-controller` also look the output too. When we run ` kompose convert --replication-controller` the output should be a file with `*-replicationcontroller.yaml` not `*-rc.yaml`.